pom.xml

```diff
@@ -56,15 +56,15 @@
         <release>8</release>
 
         <!-- Kafka and it's dependencies MUST reflect what the Kafka version uses -->
-        <version.kafka>2.5.0</version.kafka>
-        <version.kafka.scala>2.12</version.kafka.scala>
+        <version.kafka>2.6.0</version.kafka>
+        <version.kafka.scala>2.13</version.kafka.scala>
```

Comment:
Any specific reason for changing the Scala version? IIRC, I read somewhere 2.12 would be the recommended "default"?

Reply:
https://kafka.apache.org/downloads
> We build for multiple versions of Scala. This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).

Reply:
Interesting; I'm just concerned about the related change you did, as it'd might impact users of the `KafkaCluster` in tests, e.g. Quarkus.